packages/next-app/.env.example

```diff
@@ -1,3 +1,5 @@
 NEXT_PUBLIC_ALCHEMY_API_KEY=
+
 PINATA_API_KEY=
-PINATA_API_SECRET=
\ No newline at end of file
+PINATA_API_SECRET=
+NEXT_PUBLIC_PINATA_GW=https://gateway.pinata.cloud/ipfs
```

Comment:
idt it's a huge issue for this to be public, unless there's alternative gateway endpoints you see us tweaking in prod I'd leave this in the code

Reply:
this makes sense... I'll do it 😉